### PR TITLE
kubectl-create: add page

### PR DESCRIPTION
**Pull request by aditya-shrivastavv**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/kubectl-create.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# kubectl create
+
+> Create a resource from a file or from stdin..
```
Comment (gutjuri) [inline pages/common/kubectl-create.md:3]:
```suggestion
> Create a resource from a file or from `stdin`.
```

File: pages/common/kubectl-create.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+> More information: <https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#create>.
+
+- Create a resource using the resource definition file:
+
+`kubectl create -f {{path/to/file.yml}}`
+
+- Create a resource from stdin:
```
Comment (gutjuri) [inline pages/common/kubectl-create.md:10]:
```suggestion
- Create a resource from `stdin`:
```

**Review by gutjuri — COMMENTED** [review]
thanks, I have to formatting suggestions

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**